Library/Homebrew/load_path.rb

```diff
@@ -7,7 +7,20 @@
 
 $LOAD_PATH.push HOMEBREW_LIBRARY_PATH.to_s
 
+require "utils/gems"
+Homebrew.setup_gem_environment!(setup_path: false)
+
 require "vendor/bundle/bundler/setup"
+
+# Block any gem loading by bypassing rubygem's `require`.
+# Helps make sure we don't accidentally use things not in bundler's load path.
+# Bundler 2.2.7+ and non-standalone mode both do this automatically.
```

Comment:
If this code is copied from one of them: can you link to it?

Reply:
It's a simplified version of this: https://github.com/rubygems/rubygems/blob/5841761974bef324a33ef1cb650bbf8a2457805b/bundler/lib/bundler/installer/standalone.rb#L55-L63

Reply:
Cool, can that get linked inline here?